src/core/colorspace.js

```diff
@@ -606,7 +606,7 @@ var DeviceGrayCS = (function DeviceGrayCSClosure() {
     getRgbItem: function DeviceGrayCS_getRgbItem(src, srcOffset,
                                                  dest, destOffset) {
       var c = (src[srcOffset] * 255) | 0;
-      c = c < 0 ? 0 : c > 255 ? 255 : c;
+      c = Math.min(Math.max(c, 0), 255);
```

Comment:
Fixed in the new commit. I created a utility function for clamping and used that in all places for clarity. I fixed the clamping in `LabCS` as well. There is one last manual clamping part in `LabCS`, but since that uses a different kind of clamping (first to a [0,1] range and then to a [0,255] range in one step), I did not find a suitable way to convert that to use the `clamp` function and not break functionality, so I propose we leave that one place as-is.